internal/cmd/gocdk/internal/templates/init.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package templates contains string literals for use by the gocdk
+// command when creating user project files.
+package templates
+
+var InitTemplates = map[string]string{
```

Comment:
IMHO we should modify this to use the `go generate` / `static.go` approach from my other PR, as it is more flexible. Can you add a TODO and/or file an issue?

Reply:
https://github.com/google/go-cloud/issues/2015